### DISCUSSION TITLE
How to change the height of an iframe?

### DISCUSSION THREAD

**Discussion by digi-tales** [datenstrom/community#939] [Ask a question] [answered]
I tried to add an iframe, but I can only change the width. The height remains always the same. I use the stockholm-theme and the html-file is a h5p converted to html. It shows correctly but is cropped or way too small. 

`<iframe src="../media/h5p/file.html" width="100%" height="100%"> </iframe>`

**Comment by schulle4u** [comment]
You can try to enter the height in pixels, e.g. 

```
<iframe src="../media/h5p/file.html" width="100%" height="800"></iframe>
```

**Reply by digi-tales** [reply]
This works, but is there a way to always display the full height (regardless of the screen resolution)?

**Reply by schulle4u** [reply, marked as answer]
As an alternative you can use the viewport height to style the iframe, this should automatically resize it according to it's content. 

```
<iframe src="../media/h5p/file.html" width="100%" style="height: 100vh;"></iframe>
```

HTH 🙂

**Reply by digi-tales** [reply]
Thank you, there is a lot for me to learn.
